gateway/cert.go

```diff
@@ -449,8 +456,19 @@ func (gw *Gateway) certHandler(w http.ResponseWriter, r *http.Request) {
 		if certID == "" {
 			orgID := r.URL.Query().Get("org_id")
```

Comment:
Actaully I meant adding `mode` to here as it is just for `GET` requests

Reply:
moved it